### PR TITLE
[Bugfix:TAGrading] Next/prev when grading done

### LINKED ISSUE #5662: Navigating through the next/prev assigned button on TA grading throws PHP error
**Describe the bug**
When you try to navigate between students with the help of next/prev student on TA-grading page it can throw PHP error if the grading for that particular gradeable is already completed or the current user has no ungraded assigned student left to grade. 

**Expected behavior**
 If the grading is not complete and there are some ungraded assigned students left, clicking on these navigation buttons (next/prev buttons) should correctly navigate to the next/prev student. Otherwise, just show an error message that the asked student is not found.

**To Reproduce**
1. Login as `instructor` Quinn.
2. Select a gradeable for which grading is completed and hit `Regrade` button.
3. Hit grading Index button
4. Select the first student to grade and hit either next or prev double arrow button
5. See error 

**Configuration**
 - OS: macOS
 - Browser chrome, firefox

**Screenshots**
image

**Additional Context**
Error also occur if you try to access out of your assigned student list.

### PR DESCRIPTION
**Pull request by mukul-kmr-jha**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Next/prev assigned ungraded (double arrow) buttons generate error when
-  grading is completed, or 
-  you have already graded all the assigned students to you, or
-  you try to jump off the assigned student list (accessing prev assigned ungraded student from the very first assigned student or accessing the next student from the last one). 

### What is the new behavior?
Fixes #5662 
Does not throw an error (frog) and instead shows the popup that the asked student is not found.
image

### CODE REVIEW

**Comment by mukul-kmr-jha** [comment]
@bmcutler just added the changes for overflowing breadcrumbs on top of the main issue :)

**Comment by bmcutler** [comment]
This PR changes the visual appearance of the top bar and makes it unnecessarily taller.

Before:
Screen Shot 2020-07-13 at 12 40 41 AM

After:
Screen Shot 2020-07-13 at 12 39 53 AM

This happens regardless of screen width.

**Comment by bmcutler** [comment]
@jchulton  please re-review

File: site/public/css/global.css [hunk 1]
```diff
@@ -47,7 +47,7 @@ nav a > i.f {
 }
 
 #home-button {
```
Comment (jchulton) [inline site/public/css/global.css:49]: Why are there css changes? Is this due to a merge? Seems odd to have in this PR

File: site/app/templates/GlobalHeader.twig [hunk 2]
```diff
@@ -123,7 +123,7 @@
             </div>
             <div id="menu-overlay"></div>
             <nav class="flex-row shadow">
-                <div id="nav-links" style="position:relative; height:80px">
```
Comment (jchulton) [inline site/app/templates/GlobalHeader.twig:126]: These changes seem unrelated to this PR. They are also causing a small difference in display between master and this branch: the duck and Submitty are closer together on the top bar

On Master:
image

On Branch:
image

**Comment by mukul-kmr-jha** [comment]
Thanks @jchulton :)
